SecurityPkg/Include/Library/Tcg2InitEventLib.h

```diff
@@ -0,0 +1,59 @@
+/** @file
+  Initializes TCG event data structures

```

Comment:
I don't understand this library, who would use it and why?  Is it just a helper to init the caller supplied memory with HDR?   Is that hard to do and worth a library? 
Is HCRTM defined in a spec and is the plan to implement this to spec?

Reply:
At least EV_EFI_HCRTM_EVENT is defined in the TCG PFP Spec.

Reply:
I was also considering feedback that this should either be a more generic "event" library to justify the library class or moved somewhere else.